tests/1-login.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { test, expect } from '@playwright/test';
+import { LoginPage } from '../pages/login.page';
+import { AccountPage } from '../pages/account.page';
+
+
+
+
+
+test('Verify success login of existed user', async ({ page }) => {
+   test.skip(!!process.env.GITHUB_ACTIONS, 'Skip it in GitHub Actions');
+   
+  const loginPage = new LoginPage(page);
+  const accountPage = new AccountPage(page);
+  
+  await page.goto('/auth/login');
+
+  await loginPage.performLogin('customer@practicesoftwaretesting.com', 'welcome01');
```

Comment:
Optionally, how about moving the test data into a separate constant or even data file? For example:
```
const userLoginData = { 
  email: 'customer@practicesoftwaretesting.com', 
  password: 'welcome01'
}
 ```
And then
```suggestion
  await loginPage.performLogin(userLoginData.email, userLoginData.password);
```